corehq/apps/app_manager/app_schemas/app_case_metadata.py

```diff
@@ -521,7 +521,8 @@ def add_property_detail(self, detail_type, root_case_type, module_id, column):
         parts = field.split('/')
         if parts and parts[0] == 'user':
             root_case_type = USERCASE_TYPE
-            field = field.split('user/')[1]
+            if len(parts) > 1:
+                field = field.split('user/')[1]
```

Comment:
Not sure if possible, but what if `user/` occurs more than once? Is this the output you would expect?
```py
>>> field = "user/0/user/1"
>>> parts = field.split('/')
>>> assert parts and parts[0] == 'user'
>>> assert len(parts) > 1
>>> field.split('user/')[1]     # current result (assigned to field)
'0/'

>>> field.split('user/', 1)[1]  # possibly better result
'0/user/1'
```

Reply:
@millerdev i don't have enough context as to what's more ideal here...just trying to fix the 500. @proteusvacuum any ideas here?

Reply:
I think the issue I raised is separate from the issue you're fixing, so fine to proceed. A test would still be nice.